elasticdl/python/data/odps_io.py

```diff
@@ -240,6 +240,29 @@ def read_batch(self, start, end, columns=None, max_retries=3):
                 time.sleep(5)
                 retry_count += 1
 
+    def record_generator_with_retry(
+        self, start, end, columns=None, max_retries=3
+    ):
+        """Retry to read records from ODPS table by using to avoid
```

Comment:
Wrap record_generator with retry to avoid ODPS table read failure due to network instability.

Reply:
Yeah!